generator/generator.go

```diff
@@ -79,6 +79,7 @@ var (
 	includeDescriptions                  = flag.Bool("include_descriptions", false, "If set to true when generateSchema=true, the YANG descriptions will be included in the generated code artefact.")
 	enumOrgPrefixesToTrim                []string
 	ignoreUnsupportedStatements          = flag.Bool("ignore_unsupported", false, "If set to true, unsupported YANG statements are ignored.")
+	keepDeviateNotsupported              = flag.Bool("keep_deviate_notsupported", false, "If set to true, 'deviate not-supported' YANG statements are ignored, thus retaining target nodes in the generated code.")
```

Comment:
to keep the naming consistent throughout, consider using the  var name ignoreDeviateNotsupported and the flag name "ignore_deviate_notsupported"

Reply:
Done.